src/django-app/django-project/quotes/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext as _
+from shops.models import Shop
+
+class Status(models.TextChoices):
+    NEW_QUOTE = "new_quote", "New Quote"
+    ACCEPTED = "accepted", "Accepted"
+    IN_PROGRESS = "in_progress", "In Progress"
+    DONE = "done", "Done"
+    REWORK = "rework", "Rework"
```

Comment:
Sorry for the confusion on this ^.

What I meant was adding this `Status` inside of the `Quote` class.

So it would look something like: 
```python3
class Quote(models.Model):
    class Status(models.TextChoices):
        NEW_QUOTE = "new_quote", "New Quote"
        ACCEPTED = "accepted", "Accepted"
        IN_PROGRESS = "in_progress", "In Progress"
        DONE = "done", "Done"
        REWORK = "rework", "Rework"
```

You can see an example of this in `accounts.models.User` class